_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+
+### Special case: staggered and indexed leases
+Some leases explicitly mention that the rent is *indexed* on the IPC, in which case the rent will increase each year according to only the IPC, regardless of the variation of the mortgage rate. This is generally not a good deal[^indexed], as the IPC rarely decreases, and it means you cannot ask for a rent decrease when the mortgage rate decreases.
+
+Likewise, some leases are staggered: The rent is typically low in the first year, then gradually increases over a 5-year period. In that case, it is also not possible[^staggered] to ask for a rent decrease when the rates go down.
+
+## Challenging a rent increase
+If you deem a rent increase abusive, then you can choose to challenge it. In that case, you must do so within 30 days of receiving the notice, by addressing a letter to the “Commission de conciliation” of your préfecture. ASLOCA also provides a template letter. Note that for the letter to be valid, it should be signed by all the signatories of the lease, including your guarantor if you have one.
+
+This will first trigger a mediation process, between you, the landlord, and the Préfecture. The goal will be to settle the dispute so as to avoid going to court over it.
+If you go this way, then you will likely want to be a member of ASLOCA so they can provide legal advice and represent you the day of the mediation. 
+
+⚠️It may happen that the landlord simply doesn’t come to the mediation. In that case, you will be allowed by the préfecture to take the landlord to court. If so, make sure you have a case before proceeding and seek legal advice, as the process will likely last long and be very costly.
+
+In any case, after starting a procedure against your landlord, you are protected from eviction for a period of 3 years[^3ans]. A resiliation of your lease within that period may be considered as retaliation, so it is unlikely that this will happen.
+
+## Challenging the initial rent
+Similarly to a rent increase, if you spot something in the lease agreement that seems illegal, then you can challenge the initial rent within 30 days after signing the lease. The challenge is justified if one of the following conditions is met: 
+The rent is increased by more than 10% compared to the previous tenant[^10%]
+You signed the lease under duress. This does not necessarily mean you need to prove that you were held at gunpoint when signing the contract, it is justified when there are too few available apartments in your area. More specifically, if the 3-year average vacancy rate in your city is less than 1.5% (which is the case almost everywhere), that qualifies as duress. In Lausanne, the average vacancy rate is around 0.5%[^vacancy]
```

Comment:
So essentially you can always challenge the initial rent in Lausanne? And what can you demand from the landlord in such a case: that you pay exactly what the previous tenant paid, or something else? Do you know of any cases where this condition was used for a challenge?